version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.2.0"
+	Version = "0.3.0"
```

Comment:
Is this related to this change?

Reply:
Same thing

Reply:
added as part of a rebase to one of the close pr, these pr's have been open a long time.